android-agent/src/main/java/io/opentelemetry/android/SessionIdTimeoutHandler.java

```diff
@@ -40,6 +40,10 @@ final class SessionIdTimeoutHandler implements ApplicationStateListener {
         this.clock = clock;
     }
 
+    public void setSessionTimeout(final int sessionTimeout) {
+        this.SESSION_TIMEOUT_NANOS = TimeUnit.MINUTES.toNanos(sessionTimeout);
+    }
```

Comment:
We prefer immutable objects. This field should be made final and passed into the constructor.